config/templates/pages/default.xml

```diff
@@ -36,11 +36,107 @@
             <tag name="sulu.rlp"/>
         </property>
 
-        <property name="article" type="text_editor">
+        <property name="editor" type="text_editor">
             <meta>
                 <title lang="en">Article</title>
                 <title lang="de">Artikel</title>
             </meta>
         </property>
+
+        <property name="date" type="date">
```

Comment:
I would clean this file up.  I did use those properties mostly for testing some stuff.
If we want to ship out sulu with a larger default template, we also have to extend the `default.html.twig` file to render it properly.

I still think it would be nice to add blocks with a `text-with-image` block and maybe a pure `text` block, to showcase the blocks a bit